web/spec/supabase.yml

```diff
@@ -192,7 +192,11 @@ pages:
       npx openapi-typescript https://your-project.supabase.co/rest/v1/?apikey=your-anon-key --output types/supabase.ts
       ```
 
-      **Note:** Do note that your local types won't automatically stay in sync with your database, so make sure to regenerate your types after your make changes to your database.
+      Important disclaimers:
+
+      - The generator can't know if a column is specified as an array, and will thus just generate the type as `string`, even though supabase handles this automatically and returns arrays.
```

Comment:
Since the generator uses JSON API, there is no way to determine if a column is an Array. It will generate array types as `string`, even though Supabase handles this automatically and returns arrays.